packages/platform/atoms/event-types/wrappers/EventTypeWebWrapper.tsx

```diff
@@ -0,0 +1,349 @@
+
+import { useEventTypeForm } from "../hooks/useEventTypeForm";
+import { useHandleRouteChange } from "../hooks/useHandleRouteChange";
+
+const ManagedEventTypeDialog = dynamic(
+  () => import("@calcom/features/eventtypes/components/dialogs/ManagedEventDialog")
+);
+
+const AssignmentWarningDialog = dynamic(
+  () => import("@calcom/features/eventtypes/components/dialogs/AssignmentWarningDialog")
+);
+
+const EventSetupTab = dynamic(() =>
+  // import web wrapper when it's ready
+  import("@calcom/features/eventtypes/components/tabs/setup/EventSetupTab").then((mod) => mod.EventSetupTab)
+);
+
+const EventAvailabilityTab = dynamic(() =>
+  // import web wrapper when it's ready
+  import("@calcom/features/eventtypes/components/tabs/availability/EventAvailabilityTab").then(
+    (mod) => mod.EventAvailabilityTab
+  )
+);
+
+const EventTeamAssignmentTab = dynamic(() =>
+  import("@calcom/features/eventtypes/components/tabs/assignment/EventTeamAssignmentTab").then(
+    (mod) => mod.EventTeamAssignmentTab
+  )
+);
+
+const EventLimitsTab = dynamic(() =>
+  // import web wrapper when it's ready
+  import("@calcom/features/eventtypes/components/tabs/limits/EventLimitsTab").then(
+    (mod) => mod.EventLimitsTab
+  )
+);
+
+const EventAdvancedTab = dynamic(() =>
+  // import web wrapper when it's ready
+  import("@calcom/features/eventtypes/components/tabs/advanced/EventAdvancedTab").then(
+    (mod) => mod.EventAdvancedTab
+  )
+);
+
+const EventInstantTab = dynamic(() =>
+  import("@calcom/features/eventtypes/components/tabs/instant/EventInstantTab").then(
+    (mod) => mod.EventInstantTab
+  )
+);
+
+const EventRecurringTab = dynamic(() =>
+  // import web wrapper when it's ready
+  import("@calcom/features/eventtypes/components/tabs/recurring/EventRecurringTab").then(
+    (mod) => mod.EventRecurringTab
+  )
+);
+
+const EventAppsTab = dynamic(() =>
+  import("@calcom/features/eventtypes/components/tabs/apps/EventAppsTab").then((mod) => mod.EventAppsTab)
+);
+
+const EventWorkflowsTab = dynamic(
+  () => import("@calcom/features/eventtypes/components/tabs/workflows/EventWorkfowsTab")
+);
+
+const EventWebhooksTab = dynamic(() =>
+  import("@calcom/features/eventtypes/components/tabs/webhooks/EventWebhooksTab").then(
+    (mod) => mod.EventWebhooksTab
+  )
+);
+
+const EventAITab = dynamic(() =>
+  import("@calcom/features/eventtypes/components/tabs/ai/EventAITab").then((mod) => mod.EventAITab)
+);
+
+export type EventTypeWebWrapperProps = {
+  id: number;
+  isAppDir?: boolean;
+};
+
+// discriminative factor: isAppDir
+type EventTypeAppComponentProp = {
+  id: number;
+  isAppDir: true;
+  pathname: string;
+  pageRouter: null;
+};
+
+// discriminative factor: isAppDir
+type EventTypePageComponentProp = {
+  id: number;
+  isAppDir: false;
+  pageRouter: NextPageRouter;
+  pathname: null;
+};
+
+type EventTypeAppPageComponentProp = EventTypeAppComponentProp | EventTypePageComponentProp;
+
+const EventType = ({
```

Comment:
I would rename `const EventType` to `const EventTypeWeb` to better communicate that its not a re-usable event type.

Also, can we move wrapper `EventTypeWebWrapper` to very top of this file, then under it page and app wrappers and this component on very bottom for readability?